.drone.jsonnet

```diff
@@ -547,7 +547,6 @@ local integration_kubespan = Step('e2e-kubespan', target='e2e-qemu', privileged=
   WITH_CLUSTER_DISCOVERY: 'true',
   WITH_KUBESPAN: 'true',
   IMAGE_REGISTRY: local_registry,
-  WITH_CONFIG_PATCH: '[{"op": "replace", "path": "/cluster/discovery/registries/kubernetes/disabled", "value": false}]',  // use Kubernetes discovery backend
```

Comment:
don't need this here anymore?

Reply:
yep.. I think it's disabled by default for a long time now